MobileFilter/sections/specific_app.txt

```diff
@@ -812,5 +812,3 @@ c0server.jags.co.jp/jss/img/banners/$app=jp.co.jags.android.Bokumaka
 ||log.kajicam.com^
 ! Google Maps Promoted Pins
 ||mobilemaps-pa.googleapis.com/google.internal.mothership.maps.mobilemaps.ads.v1.MobileMapsAdsService/ListPromotedPinAds^
-! WeChat
-||extshort.weixin.qq.com/mmtls^$app=com.tencent.mm
```

Comment:
Why removed?

Reply:
this rule may affect the other features of this app